whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapObjectMapperTest.java

```diff
@@ -397,6 +402,38 @@ public void organisation() {
         assertThat(result.getPort43(), is("whois.ripe.net"));
 
         assertThat(result.getLang(), is("DK"));
+
+        final List<Notice> notices = result.getNotices();
+
+        MatcherAssert.assertThat(notices.get(0).getTitle(), is("Multiple language found"));
+        MatcherAssert.assertThat(notices.get(0).getDescription().get(0), is("There are multiple language DK,EN in ORG-AC1-TEST, but only the first language DK was returned."));
+    }
+
+    @Test
+    public void organisation_should_not_have_notice_for_language() {
+        final Entity result = (Entity) map(RpslObject.parse("" +
+                "organisation:   ORG-AC1-TEST\n" +
+                "org-name:       Acme Carpets\n" +
+                "org-type:       OTHER\n" +
+                "address:        Singel 258\n" +
+                "e-mail:         bitbucket@ripe.net\n" +
+                "descr:          Acme Carpet Organisation\n" +
+                "remark:         some remark\n" +
+                "phone:          +31 1234567\n" +
+                "fax-no:         +31 98765432\n" +
+                "geoloc:         52.375599 4.899902\n" +
+                "language:       DK\n" +
+                "admin-c:        TP1-TEST\n" +
+                "abuse-c:        ABU-TEST\n" +
+                "mnt-by:         FRED-MNT\n" +
+                "source:         TEST"));
+
+        assertThat(result.getLang(), is("DK"));
+
+        final List<Notice> notices = result.getNotices();
+
+        assertThat(notices, hasSize(1));
+        assertNull(notices.get(0));
```

Comment:
Why is there a null notice?

Reply:
mapCommons method adds an empty notice apparently.
rdapResponse.getNotices().add(noticeFactory.generateTnC(requestUrl));
this line adds a null notice.

Reply:
Can that be fixed? Strange that generateTnC returns null?

Reply:
In reality generateTnc will never return null, integrationTest always have TnC, Its Junit that we didn't mock the call to method generateTnc to return a notice. I will change the unit test case for it. I don't think mapCommons needs to have a null check for notice?

Reply:
Ok thanks